pkg/model/components/etcdmanager/model.go

```diff
@@ -383,6 +383,7 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster kops.EtcdClusterSpec, instance
 			config.VolumeTag = []string{
 				fmt.Sprintf("kubernetes.io/cluster/%s=owned", b.Cluster.Name),
 				awsup.TagNameEtcdClusterPrefix + etcdCluster.Name,
+				awsup.TagNameRolePrefix + "control-plane=1",
```

Comment:
Changes to this file feel incomplete:
1. new `control-plane` role is only added to some cloud providers
2. probably we want to remove the old `master` tag in this case, as `etcd-manager` should only look for the new one.
Any thoughts?

Reply:
Wouldn't removing the old master tag break kops downgrades?

Reply:
This is not about adding / removing the tag. This just generates a list of tags that etcd-manager should match for a volume. By removing "master", it should just ignore the existence of that tag.

Reply:
I believe additional cloud providers is a separate PR. I only did the ones that were obvious.

If it's the query, I can remove the old tag out of `EtcdManagerBuilder`.

Reply:
I switched Azure over. For GCE I don't know enough about the tagging restrictions

Reply:
That would be an excellent topic for a follow-up PR. There are a lot of opportunities in the code base to update the terminology. I was trying to limit this PR to things that derive directly from the IG role, though I unfortunately got some scope creep into etc volumes on providers that don't directly derive from the IG role.

Reply:
👍